framework/bundles/org.jboss.tools.rsp.server/src/main/java/org/jboss/tools/rsp/server/model/RemoteEventManager.java

```diff
@@ -129,5 +110,26 @@ public void vmChanged(PropertyChangeEvent event) {
 		// TODO Auto-generated method stub
 		
 	}
+
+
+	private VMDescription getDescription(IVMInstall vmi) {
```

Comment:
you want to keep the vminstall infrastructure? I'd think that we should get rid of anything that we dont use, remove the clutter (unused code is clutter).